.changeset/flat-oranges-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@elastic/behavioral-analytics-tracker-core": major
```

Comment:
a major will result in v3 of core, so I suggest because it hasn't been published before, to keep it as a patch